packages/story-editor/src/components/publishModal/content/storyPreview.js

```diff
@@ -246,12 +285,25 @@ const StoryPreview = () => {
                     data-testid="story_preview_logo"
                   />
                 )}
-                {ENABLE_EDIT_FEATURED_MEDIA && hasUploadMediaAction && (
-                  <EditFeaturedMedia
-                    aria-label={__('Edit Publisher Logo', 'web-stories')}
-                  >
-                    <Icons.Pencil aria-hidden="true" />
-                  </EditFeaturedMedia>
+                {hasUploadMediaAction && (
+                  <StyledMedia
+                    onChange={handleChangePoster}
+                    title={__('Select as poster image', 'web-stories')}
+                    buttonInsertText={__(
+                      'Select as poster image',
+                      'web-stories'
+                    )}
+                    type={allowedImageMimeTypes}
+                    ariaLabel={__('Poster image', 'web-stories')}
+                    onChangeErrorText={posterErrorMessage}
+                    imgProps={featuredMedia}
+                    canUpload={hasUploadMediaAction}
+                    variant={MEDIA_VARIANTS.NONE}
+                    cropParams={{
+                      width: 640,
+                      height: 853,
+                    }}
```

Comment:
Would be nice to centralize this but currently lives in one other place: https://github.com/GoogleForCreators/web-stories-wp/blob/928311c162f7d7f0454364755e689ef099badb33/packages/wp-story-editor/src/components/documentPane/publish/publish.js#L323-L325

Reply:
🤔  Unsure the best place to centralize these. Feels like configs for the actual story so it should be in the story-editor and then imported to wp-story-editor? Def out of scope here, curious where these should call their home eventually.

Reply:
Yeah, something we can tackle later. I'd say it's a config too

Reply:
yea config makes sense to me too